Bottom: a9529e960b3f21af7a20e8284f561073377cc986
Top:    acb5508d62534c0db9eb3ef2553d1c264ed84b20
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 12:36:06 +0000

Refresh of add-void-userdata-member-to

---

diff --git a/src/polyarea.h b/src/polyarea.h
index 76b724a..bcb2152 100644
--- a/src/polyarea.h
+++ b/src/polyarea.h
@@ -159,6 +159,7 @@ struct POLYAREA
     PLINE *contours;
     rtree_t *contour_tree;
     POLYPARENTAGE parentage;
+    void *user_data; /* To be used at will by the owner of this polygon */
 };
 
 void poly_dump (POLYAREA *p);
diff --git a/src/polygon1.c b/src/polygon1.c
index 1372197..2c75fa4 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -3901,6 +3901,7 @@ poly_Init (POLYAREA * p)
   p->contours = NULL;
   p->contour_tree = r_create_tree (NULL, 0, 0);
   p->parentage = no_parentage;
+  p->user_data = NULL;
 }
 
 POLYAREA *
